discovery-provider/src/trending_strategies/base_trending_strategy.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+from src.trending_strategies.trending_type_and_version import TrendingType, TrendingVersion
+
+class BaseTrendingStrategy(ABC):
+    def __init__(self, trending_type, version):
+        self.trending_type = TrendingType.UNDERGROUND_TRACKS
```

Comment:
super nit, maybe default should be normal trending not underground

Reply:
oh this was completely wrong! nice catch